authz.go

```diff
@@ -157,6 +159,7 @@ var AllResourceTypes = []ResourceType{
 	NotificationEndpointResourceType, // 15
 	ChecksResourceType,               // 16
 	DBRPResourceType,                 // 17
+	NotebooksResourceType,            // 18
 	// NOTE: when modifying this list, please update the swagger for components.schemas.Permission resource enum.
```

Comment:
The list in question: https://github.com/influxdata/openapi/blob/ec6019622d60053d4761809cb2773ba0e28b363a/contracts/oss.yml#L12208

Since I'm not doing this as part of this PR (and won't be doing it immediately), I created an issue to include in the Notebooks epic so I don't forget to do it: https://github.com/influxdata/influxdb/issues/21491